eox_core/api/v1/views.py

```diff
@@ -200,6 +203,53 @@ def patch(self, request, *args, **kwargs):
         return Response(serialized_user.data)
 
 
+class EdxappReplaceUsername(UserQueryMixin, APIView):
```

Comment:
Same feedback as in #138, maybe use a backend and the existing user endpoint

Reply:
Done